### ISSUE TITLE
Makefile for the basic build/run commands

### ISSUE THREAD

**Issue by andrewgy8** [apoclyps/my-dev-space#431]
**Is your feature request related to a problem? Please describe.**
The problem is the copypasta necessary to get up and running.

**Describe the solution you'd like**
A nice Makefile in the root of the project where commands could be defined. You execute `make env-seed` which would in turn run the equivalent of:

```
$ docker-compose -f docker-compose-dev.yml up -d
$ docker-compose -f docker-compose-dev.yml run users-service python manage.py recreate_db
$ docker-compose -f docker-compose-dev.yml run users-service python manage.py seed_db
$ docker-compose -f docker-compose-dev.yml run events-service python manage.py recreate_db
```

Other commands could be `make test` and `make lint`.  There are others that could be included, but this would be a good start, and further commands can be added later.

**Describe alternatives you've considered**
I would say there are alternatives, but `make` seems to be the best system agnostic solution.  

**Additional context**
This would make the project `plug-and-play` for any developer or user.

**Comment by apoclyps** [comment]
Thank you @andrewgy8 for submitting this issue. Adding a `Makefile` has been something I've been considering and delaying as it provides an additional layer of abstraction on top of docker-compose and requires an additional project dependency; 

However, I agree the project setup should be as smooth and easy to set up as possible and in this instance, a Makefile is much simpler than running a bunch of commands or trying to find a `./tests.sh` a script in the repo. Additionally, it will make linting all projects much easier.

Would you like to submit a PR for this issue (as part of Hacktoberfest)?

**Comment by Warlord77** [comment]
may I work on this ?

**Comment by apoclyps** [comment]
@Warlord77 yes, of course. I've sent you an invite. Once you accept I'll assign the issue to you.

**Comment by andrewgy8** [comment]
@apoclyps I was going to submit a PR, but if @Warlord77 has got it, then go for it!

**Comment by apoclyps** [comment]
@andrewgy8 I've assigned the issue across to yourself as the invitation wasn't accepted to assign the issue. Issues assignment is on a first come, first serve basis. Until the issue is assigned, it's free for anyone to pick up.

@Warlord77, if you are still interested in working on something. Have another look at the [issues](https://github.com/apoclyps/my-dev-space/issues) or direct message me on Twitter @apoclyps and we can come up with something to fit your skillset 👍

**Comment by apoclyps** [comment]
@andrewgy8 any update on this ticket? Is there anything I can help with?

**Comment by andrewgy8** [comment]
@apoclyps Yep, ill try to have a PR in this evening.

**Comment by andrewgy8** [comment]
The one thing Im having trouble with is getting the local environment set up.  I run in to 502 Bad gateway errors from nginx.  It seems like the client service defined in the docker-compose does not spin up properly.  

From `docker logs nginx`:

> 2018/10/18 12:57:31 [error] 9#9: *1 connect() failed (113: No route to host) while connecting to upstream, client: 172.31.0.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.31.0.7:3000/", host: "localhost"
172.31.0.1 - - [18/Oct/2018:12:57:31 +0000] "GET / HTTP/1.1" 502 157 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.13; rv:62.0) Gecko/20100101 Firefox/62.0" "-" 

Any ideas?

**Comment by apoclyps** [comment]
@andrewgy8 I believe you may already have solved the above issue however for future contributors; this error occurs whilst the client is still being built and set up. nginx is configured and available on `localhost` usually a minute before the frontend client is up. Typically waiting a minute allows the client enough time to come up; alternatively dropping the `-d` on the up command for running the service provides context on the service starting up.

**Comment by apoclyps** [comment]
@andrewgy8 thanks again for this PR; closed by #478
